packages/cli/src/lib/snapshot/reportPreviewer/reportPreviewer.spec.ts

```diff
@@ -23,34 +23,11 @@ describe('ReportViewer', () => {
     test
       .stdout()
       .stderr()
-      .do(() => {
-        viewer.display();
+      .do(async () => {
+        await viewer.display();
       })
-      .it(
-        'should print a report section with the resources in error',
-        (ctx) => {
-          expect(ctx.stdout).toContain('Error Report:');
-          expect(ctx.stdout).toContain('7 resources in error');
-        }
-      );
-
-    test
-      .stdout()
-      .stderr()
-      .do(() => {
-        viewer.display();
-      })
-      .it('should not print more than 5 errors per resources', (ctx) => {
-        expect(ctx.stdout).toContain(
-          dedent`
-          Fields
-            • RESOURCE_ALREADY_EXISTS: Field foo already exists.
-            • RESOURCE_ALREADY_EXISTS: Field bar already exists.
-            • RESOURCE_ALREADY_EXISTS: Field dsads already exists.
-            • RESOURCE_ALREADY_EXISTS: Field fdww already exists.
-            • RESOURCE_ALREADY_EXISTS: Field csad already exists.
-            (2 more errors)`
-        );
+      .it('should print the same report', (ctx) => {
+        expect(ctx.stdout).toMatchSnapshot();
```

Comment:
I don't remember if we knew about snapshots back then, I reckon not.
This is the perfect use case for them, so here they go.